aws-android-sdk-iot/src/test/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManagerTest.java

```diff
@@ -33,7 +33,7 @@
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 
-import static com.ibm.icu.impl.Assert.fail;
+import static org.junit.Assert.fail;
```

Comment:
import statements should be alphabetized for future reference, but this project does not enforce that so let's get the important change in first.